package.json

```diff
@@ -9,7 +9,9 @@
     "local": "npm-run-all --parallel dev rollup:watch",
     "build": "rollup -c && next build && next export",
     "start": "next start",
-    "test": "jest",
+    "test": "npm run lint && jest",
+    "lint": "standard",
```

Comment:
looking ahead, if we do get around to implementing stylelint for scss files, can we use the same command or should we split them to lint:js lint:scss.
your call, I'm happy either way

Reply:
good question. maybe `lint` does it all, by calling `lint:js` etc, in the same way `test` does it all, but the developer can, if they choose, run them separately.

Reply:
cool